src/main/java/org/openstreetmap/atlas/generator/PbfLocator.java

```diff
@@ -124,12 +119,19 @@ public PbfLocator(final PbfContext pbfContext, final Map<String, String> spark)
      */
     public Iterable<LocatedPbf> pbfsCovering(final MultiPolygon multiPolygon)
     {
-        logger.trace("Seeking tiles for MultiPolygon {}", multiPolygon.toSimpleString());
+        if (logger.isTraceEnabled())
```

Comment:
Is this a Sonar thing?

Reply:
Yes, to avoid calling `multiPolygon.toSimpleString()` if not necessary.